data/jhu.py

```diff
@@ -0,0 +1,254 @@
+    "csse_covid_19_data/csse_covid_19_time_series/"
+    "time_series_covid19_confirmed_global.csv"
+)
+
+# URL to JHU deaths time series.
+DEATHS_URL = (
+    "https://github.com/CSSEGISandData/COVID-19/raw/{}/"
+    "csse_covid_19_data/csse_covid_19_time_series/"
+    "time_series_covid19_deaths_global.csv"
+)
+
+# URL to JHU recoveries time series
+RECOVERED_URL = (
+    "https://github.com/CSSEGISandData/COVID-19/raw/{}/"
+    "csse_covid_19_data/csse_covid_19_time_series/"
+    "time_series_covid19_recovered_global.csv"
+)
+
+# Feature ID for JHU global source data
+#JHU_GLOBAL_SOURCE_ID = "c0b356e20b30490c8b8b4c7bb9554e7c"
+JHU_GLOBAL_SOURCE_ID = (
+    "https://services1.arcgis.com/0MSEUqKaxRlEPj5g/arcgis/rest/services/"
+    "ncov_cases/FeatureServer/0/query?where=1%3D1&objectIds=&time=&geometry=&"
+    "geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&"
+    "resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&"
+    "outFields=OBJECTID%2C+Province_State%2C+Country_Region%2C+Last_Update%2C+Lat%2C+Long_%2C"
+    "+Confirmed%2C+Recovered%2C+Deaths%2C+Active%2C+Admin2%2C+FIPS%2C+Combined_Key&"
+    "returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&"
+    "maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&"
+    "returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&"
+    "returnQueryGeometry=false&returnDistinctValues=false&cacheHint=false&"
+    "orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&"
+    "resultOffset=&resultRecordCount=&returnZ=false&returnM=false&"
+    "returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pgeojson&token="
+)
+
+# The date at the time of execution. We choose midnight in the US/Pacific timezone,
+# but then convert to UTC since that is what AGOL expects. When the feature layer
+# is viewed in a dashboard it is converted back to local time.
+# date = pd.Timestamp.now(tz="US/Pacific").normalize().tz_convert("UTC")
+
+
+def parse_columns(df):
+    """
+    quick helper function to parse columns into values
+    uses for pd.melt
+    """
+    columns = list(df.columns)
+
+    id_vars, dates = [], []
+
+    for c in columns:
+        if c.endswith("20"):
+            dates.append(c)
+        else:
+            id_vars.append(c)
+    return id_vars, dates
+
+
+def coerce_integer(df):
+    """
+    Coerce nullable columns to integers for CSV export.
+
+    TODO: recent versions of pandas (>=0.25) support nullable integers.
+    Once we can safely upgrade, we should use those and remove this function.
+    """
+
+    def integrify(x):
+        return int(float(x)) if not pd.isna(x) else None
+
+    cols = [
+        "number_of_cases",
+        "number_of_deaths",
+        "number_of_recovered",
+    ]
+    new_cols = {c: df[c].apply(integrify, convert_dtype=False) for c in cols}
+    return df.assign(**new_cols)
```

Comment:
Can we try to use the `"Int64"` dtype rather than this hack, now that we are no longer stuck on pandas 0.24? Could be in a follow-up, but as long as we are moving off of the old airflow+Esri stack, we can remove some workarounds.

Reply:
Yep, Int64 works!